poll/public/js/poll.js

```diff
@@ -29,6 +29,9 @@ function PollUtil (runtime, element, pollType) {
             // studio.
             radio = $(selector, element);
             var choice = radio.val();
+            var thanks = $('.poll-voting-thanks', element);
+            thanks.addClass('poll-hidden');
+            thanks.removeAttr('style');
```

Comment:
It might be good to add a comment to explain why removing the style attribute is necessary.